cluster-api-gcp/deploy/deploy_helper.go

```diff
@@ -293,3 +299,21 @@ func (d *deployer) waitForApiserver(master string, timeout time.Duration) error
 	}
 	return err
 }
+
+// Make sure the default service account in kube-system namespace exists.
+func (d *deployer) waitForServiceAccount(timeout time.Duration) error {
+	client, err := apiutil.NewKubernetesClient(d.configPath)
+	if err != nil {
+		return err
+	}
+	startTime := time.Now()
+
+	for time.Now().Sub(startTime) < timeout {
+		_, err := client.CoreV1().ServiceAccounts("kube-system").Get("default-foo", metav1.GetOptions{})
```

Comment:
add kube-system and service acct name as constant?

Reply:
Done.